main/Stare_Dobre_Malzenstwo/piosenka_dla_juniora_i_jego_gitary.tex

```diff
@@ -0,0 +1,61 @@
+\tytul{Piosenka dla juniora i jego gitary}{sł. Edward Stachura, muz. Krzysztof Myszkowski}{Stare Dobre Małżeństwo}
+\begin{text}
+	Gdy pokłócisz się z dziewczyną\\
```

Comment:
Te linie zaczynają się tabulatorem zamiast spacjami, dobrze by było to przerobić.

Protip: w notepadzie++ zaznaczasz całą sekcję, shift+tab usuwa wcięcia a później sam TAB je dodaje, już ze spacjami jeśli masz dobrze ustawionego :)